providers/gcp/resources/gcp.lr

```diff
@@ -1490,6 +1490,8 @@ private gcp.project.gkeService.cluster @defaults("name description zone status c
   identityServiceConfig dict
   // Configuration options for the NetworkPolicy feature
   networkPolicyConfig dict
+  // Configuration for Release Channel feature
+  releaseChannel dict
```

Comment:
do we plan to extend this or does this only hold a string? if so, its probably best to just make this a `releaseChannel string` in that case